docs/index.md

```diff
@@ -0,0 +1,55 @@
+<img src="assets/images/architecture.svg">
+
+<a name="quick"></a>
+## Quick Start
+
+
+```bash
+bash <(curl -s https://repo.hops.works/installer/latest/hopsworks-installer.sh)
+```
+Recommended minimum [specification](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-installer.html#requirements).
+
+The [Hopsworks Installer](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-installer.html) takes roughly 1-2 hrs to complete, depending on your bandwidth.
+
+
+### Installer Quick Start with Azure CLI or GCP CLI
+
+```python
+bash <(curl -s https://repo.hops.works/installer/latest/hopsworks-cloud-installer.sh)
+```
+If you have the Azure or GCP CLI utilities installed (on a Linux machine), then the [Hopsworks Cloud Installer](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-cloud-installer.html) (Hopsworks-cloud-installer.sh) will both install Hopsworks and provision the VMs in one command.
+
+## Development and Operational ML on Hopsworks
+You can develop and run Python, Spark, and Flink applications on Hopsworks - in Jupyter notebooks, as jobs, or even notebooks as jobs. You can build production pipelines with the bundled Airflow, and even run ML training pipelines with GPUs in notebooks on Airflow. You can train models on as many GPUs as are installed in a Hopsworks cluster and easily share them among users.
+
+## Who’s behind Hopsworks?
+Hopsworks started as an open-source collaborative project at [KTH University](https://www.kth.se/en), [RISE](https://www.ri.se/en), and has more recently been taken on by [Logical Clocks](https://www.logicalclocks.com/). Several funding bodies have helped contribute to its development including: [European Commission (FP7, H2020)](https://ec.europa.eu/), [EIT](https://eit.europa.eu/), [SSF](https://strategiska.se/), [Vinnova](https://www.vinnova.se/) and [Celtic-Next](https://www.celticnext.eu/).
+
+
+## Open-Source
+Hopsworks is available under the AGPL-V3 license. In plain English this means that you are free to use Hopsworks and even build paid services on it, but if you modify the source code, you should also release your changes and any systems built around it as AGPL-V3.
+
+
+## Join the community
+-  Ask questions and give us feedback in the [Hopsworks Community](https://community.hopsworks.ai/)
+- Follow us on [Twitter](https://twitter.com/hopsworks)
+- Check out all our latest [product releases](https://github.com/logicalclocks/hopsworks/releases)
+
+
+## Contribute
+We are building the most complete and modular ML platform available in the market and we count on your support to continuously improve Hopsworks. Feel free to [give us suggestions](https://github.com/logicalclocks/feature-store-api), [report bugs](https://github.com/logicalclocks/feature-store-api/issues) and [add features to our library](https://github.com/logicalclocks/feature-store-api) anytime.
```

Comment:
are these links correct?

Reply:
I fixed two of them to point to hopsworks instead.